hrjobcontract/js/src/job-contract/controllers/contract.js

```diff
@@ -249,9 +250,9 @@ define([
                                 (dateEffectiveRevisionCurrent > dateToday &&
                                 dateEffectiveRevisionCreated <= dateEffectiveRevisionCurrent);
 
+
```

Comment:
On the other hand, I think you don't need this additional blank line here. Just one is enough :D